build.gradle

```diff
@@ -137,7 +137,7 @@ runIde {
 
 intellij {
     // For a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
-    version = '2023.3.4'
+    version = '2024.1'
```

Comment:
We should target 2024.1.1 which seems to resolve the `runIde` crash that was occurring on Windows when opening a new terminal.

Reply:
OK